<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="baudRateComboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Custom</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="customBaudRateLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter custom rate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="fileFormatComboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Save as .txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Save as .csv</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="refreshPortsButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh Ports</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>271</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
